virtcontainers/network.go

```diff
@@ -212,6 +215,11 @@ func (endpoint *VirtualEndpoint) Type() EndpointType {
 	return endpoint.EndpointType
 }
 
+// PciAddr returns the PCI address of the endpoint.
+func (endpoint *VirtualEndpoint) PciAddr() string {
```

Comment:
`s/PciAddr/PCIAddr` ?

Reply:
That will be same with field name and makes an error.